### ISSUE TITLE
[Suggestion] Allow changing horseshoe loot spawn rate and add optional recipes via config

### ISSUE THREAD

**Issue by pajicadvance** [HyperPigeon/Horseshoes#12]
The default spawn rates are a bit on the higher side, it would be useful to have a mod configuration to change the spawn rate, or disable spawning in loot altogether and use a crafting recipe instead.

**Comment by Beez3-A** [comment]
I totally agree with this, the spawn rates are actually ridiculously high. It's so high I cant use this mod. 

great idea but lacks those final touches. also the items are too big in the GUI. 

again great mod but I agree with this suggestion 1000%

**Comment by HyperPigeon** [comment]
I've reduced the spawn rate as a temporary stop-gap. I will add a proper config in the future. 

Also, you can add a crafting recipe using a datapack as temporary measure as well.
